chall07/rpehkone.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/python3
+
+import sys
+
+def usage():
+	print("usage: " + sys.argv[0] + " <1-9 squared_rows...>")
+	exit()
```

Comment:
instead of print + exit, you can directly give exit an argument message to print. 

exit("your message")